wireguard.sh

```diff
@@ -58,7 +58,7 @@ latest_release_for() {
 	# > The created_at attribute is the date of the commit used for the
 	# > release, and not the date when the release was drafted or published.
 	curl -sSL https://api.github.com/repos/${WIREGUARD_REPO}/releases \
-		| jq -r --arg version "wireguard-${board}" \
+		| jq -r --arg version "${board}" \
```

Comment:
I'm thinking this `jq` expression will no longer work correctly for devices that have `v1` / `v2` versions available. We probably need to default to setting board to something like `ugw3-v1` by default, and changing it to `v2` if the kernel version requires it.

Reply:
the naming has changed, so line 326 must be changed from
```
                BOARD=v2.0-$BOARD 
```
to
```
                BOARD=$BOARD-v2
```

that did work for me